.github/workflows/docker-hub-develop.yml

```diff
@@ -26,10 +31,12 @@ jobs:
 
       # Needed for multi platform builds
       - name: Set up QEMU
-        uses: docker/setup-qemu-action@v3.0.0
+        uses: docker/setup-qemu-action@v3
+        with:
+          platforms: arm64
```

Comment:
Just double checking but this doesn't mean that we only build for arm64 right?

Reply:
No this just means it only needs to install the arm64 qemu stuff. It should natively build amd64 stuff still with this from how I understand the documentation.

Reply:
guess we'll find out, but do you think we should state it explicitly just to be sure?

Reply:
I can add a comment in a moment :)

Reply:
Turns out you were right and this is wrong. Fixing it